### PR TITLE
Amend analytics text as per client request - prod release

### PR DESCRIPTION
**Pull request by jkeasley**
Amended the analytics.txt at clients request, this is a cherry-pick of the commit which has already been deployed to dev to allow the change to be deployed to prod

### CODE REVIEW

**Comment by snim2** [comment]
Is there a reason for cherry-picking this one commit, rather than merging `dev->prod`?

File: app/templates/analytics.txt [hunk 1]
```diff
@@ -1 +1 @@
-GooGhywoiu9839t543j0s7543uw1 - pls add steven.bagley@nhs.net to GA account UA-137452494 with ‘Manage Users and Edit’ permissions - date 8 December 2022.
\ No newline at end of file
+GooGhywoiu9839t543j0s7543uw1.  Please add steven.bagley@nhs.net to GA account 137452494 with “Administrator” permissions - date 16th December 2022.
```
Comment (snim2) [inline app/templates/analytics.txt:1]: I don't know how picky Google is about this format, but there are two spaces after the `.`. If there's a human in the loop with this process then it certainly isn't worth changing.

**Comment by jkeasley** [comment]
@snim2 Yep, the dev instance currently has the new alert banners merged into it, which haven't been signed off by NHSX as of yet

**Comment by snim2** [comment]
> @snim2 Yep, the dev instance currently has the new alert banners merged into it, which haven't been signed off by NHSX as of yet

Oh gosh, OK. Let's remember to merge this back to `develop` then!

**Review by snim2 — APPROVED** [review]

File: app/templates/analytics.txt (hunk 1 above)
Comment (jkeasley) [reply app/templates/analytics.txt:1]: I assume that given the google help for analytics.txt also has two spaces at this point that this should be ok ;)

**Pull request merged**